### PR TITLE
Add formats parameter to aenetworks TP_SMIL_QUERY items

### LINKED ISSUE #4093: [aenetworks]: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

United States

### Description

Specific video from a site that other videos work.
Playable in browser with DRM disabled.

### Verbose log

```shell
% yt-dlp -v "${URL}" -o "alone.s09e03.mp4" 
[debug] Command-line config: ['-v', 'https://play.history.com/shows/alone/season-9/episode-3', '-o', 'alone.s09e03.mp4']
[debug] User config "/home/user/.config/yt-dlp.conf": ['--ap-mso', 'Verizon', '--ap-username', 'PRIVATE', '--ap-password', 'PRIVATE']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d52355]
[debug] Python version 3.10.5 (CPython 64bit) - Linux-5.18.3-arch1-1-x86_64-with-glibc2.35
[debug] Checking exe version: ffprobe -bsfs
[debug] Checking exe version: ffmpeg -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1
[debug] Optional libraries: Cryptodome-3.12.0, certifi-2021.10.08, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [aenetworks] Extracting URL: https://play.history.com/shows/alone/season-9/episode-3
[aenetworks] /shows/alone/season-9/episode-3: Downloading JSON metadata
[aenetworks] 2041648195915: Downloading JSON metadata
[aenetworks] 2041648195915: Downloading hls_high_ak SMIL data
[aenetworks] 2041648195915: Downloading high_video_s3 SMIL data
[aenetworks] 2041648195915: Downloading hls_high_fastly SMIL data
ERROR: [aenetworks] shows/alone/season-9/episode-3: None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 642, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/aenetworks.py", line 169, in _real_extract
    return self._extract_aetn_info(domain, 'canonical', '/' + canonical, url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/aenetworks.py", line 88, in _extract_aetn_info
    info.update(self._extract_aen_smil(media_url, video_id, auth))
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/aenetworks.py", line 61, in _extract_aen_smil
    raise last_e
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/aenetworks.py", line 51, in _extract_aen_smil
    tp_formats, tp_subtitles = self._extract_theplatform_smil(
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/theplatform.py", line 45, in _extract_theplatform_smil
    raise ExtractorError(

-------------------- Working Video ---------------
% yt-dlp -v "${URL}" -o output.mp4                          
[debug] Command-line config: ['-v', 'https://play.history.com/shows/alone/season-9/episode-4', '-o', 'output.mp4']
[debug] User config "/home/user/.config/yt-dlp.conf": ['--ap-mso', 'Verizon', '--ap-username', 'PRIVATE', '--ap-password', 'PRIVATE']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d52355]
[debug] Python version 3.10.5 (CPython 64bit) - Linux-5.18.3-arch1-1-x86_64-with-glibc2.35
[debug] Checking exe version: ffprobe -bsfs
[debug] Checking exe version: ffmpeg -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1
[debug] Optional libraries: Cryptodome-3.12.0, certifi-2021.10.08, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [aenetworks] Extracting URL: https://play.history.com/shows/alone/season-9/episode-4
[aenetworks] /shows/alone/season-9/episode-4: Downloading JSON metadata
[aenetworks] 2044162627627: Downloading JSON metadata
[debug] Loading ap-mvpd.HISTORY from cache
[aenetworks] 2044162627627: Retrieving Media Token
[aenetworks] 2044162627627: Downloading hls_high_ak SMIL data
[aenetworks] 2044162627627: Downloading m3u8 information
[aenetworks] 2044162627627: Downloading high_video_s3 SMIL data
[aenetworks] 2044162627627: Downloading m3u8 information
[aenetworks] 2044162627627: Downloading hls_high_fastly SMIL data
[aenetworks] 2044162627627: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 2044162627627: Downloading 1 format(s): hls-2695-2
[debug] Invoking hlsnative downloader on ...
```

### PR DESCRIPTION
**Pull request by jacobtruman**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

Fix bug with certain aenetworks video files returning the error `None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences `

Fixes #4093

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

**Comment by pukkandan** [comment]
Why are we requesting DRM'd formats?

**Comment by jacobtruman** [comment]
@pukkandan a valid question to which I do not have a great answer. I found this ya what is used in the videos I came across that suffered from this issue.

**Comment by pukkandan** [comment]
What happens if you set it to just `M3U+none,MPEG-DASH+none,MPEG4,MP3`?

**Comment by jacobtruman** [comment]
> What happens if you set it to just `M3U+none,MPEG-DASH+none,MPEG4,MP3`?

That works as well - good call

File: yt_dlp/extractor/aenetworks.py [hunk 1]
```diff
@@ -29,16 +29,20 @@ class AENetworksBaseIE(ThePlatformIE):
...
+            'switch': 'hls_high_ak',
+            'formats': query_formats,
         }, {
-            'assetTypes': 'high_video_s3'
+            'assetTypes': 'high_video_s3',
+            'formats': query_formats,
         }, {
             'assetTypes': 'high_video_s3',
             'switch': 'hls_high_fastly',
+            'formats': query_formats,
```
Comment (pukkandan) [inline yt_dlp/extractor/aenetworks.py:45]:
```suggestion
        query = {
            'mbr': 'true',
            'formats': 'M3U+none,MPEG-DASH+none,MPEG4,MP3',
        }
        if auth:
            query['auth'] = auth
        TP_SMIL_QUERY = [{
            'assetTypes': 'high_video_ak',
            'switch': 'hls_high_ak',
        }, {
            'assetTypes': 'high_video_s3',
        }, {
            'assetTypes': 'high_video_s3',
            'switch': 'hls_high_fastly',
```

Comment (jacobtruman) [reply yt_dlp/extractor/aenetworks.py]: done

**Pull request merged**